app/static/js/shared_list.js

```diff
@@ -0,0 +1,44 @@
+var items_num = 1;
```

Comment:
Prefer not to have a global variable which hold state. You can use data-attributes to store states, or just count the using `document.querySelectorAll` with `.length`